app/internal/languages.py

```diff
@@ -59,27 +55,7 @@ def set_ui_language(language: str = None) -> None:
 #     return config.WEBSITE_LANGUAGE
 
 
-def _get_language_directory() -> str:
-    """Returns the language directory relative path."""
-    language_dir = LANGUAGE_DIR
-    if Path(LANGUAGE_DIR_TEST).is_dir():
-        # If running from test, change dir path.
-        language_dir = LANGUAGE_DIR_TEST
-    return language_dir
-
-
-def _get_supported_languages(
-        language_dir: str = _get_language_directory()
-) -> Iterator[str]:
-    """Returns a generator of supported translation languages codes.
-
-    Args:
-        language_dir: Optional; The path of the language directory.
-            Defaults to the return value of _get_language_directory().
-
-    Returns:
-        A generator expression of the supported translation languages codes.
-    """
-
-    paths = [Path(f.path) for f in os.scandir(language_dir) if f.is_dir()]
+def _get_supported_languages() -> Iterator[str]:
+    """Returns a generator of supported translation languages codes."""
+    paths = [Path(f.path) for f in os.scandir(LOCALES_PATH) if f.is_dir()]
```

Comment:
I'd go with generator expression here (so the memory won't store many paths in a single time)

Reply:
Isn't it now?
```
    paths = [Path(f.path) for f in os.scandir(LOCALES_PATH) if f.is_dir()]
    return (language.name for language in paths)
```

Reply:
fixed.